### PR TITLE
refactor: move PackageJson to deno_config

### PR DESCRIPTION
**Pull request by dsherret**
~~Waiting on https://github.com/denoland/deno_config/pull/60~~

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1106,7 +1106,7 @@ dependencies = [
  "glibc_version",
  "glob",
  "ignore",
- "import_map 0.20.0",
+ "import_map",
```
Comment (dsherret) [inline Cargo.lock:1109]: Oh, whoops. We accidentally have two import_map deps in main.

File: cli/npm/byonm.rs [hunk 2]
```diff
@@ -181,68 +244,29 @@ impl CliNpmResolver for ByonmCliNpmResolver {
...
-        .root_node_modules_dir
-        .parent()
-        .unwrap()
-        .join("package.json"),
-    )
+    bail!(
+      concat!(
+        "Could not find \"{}\" in a node_modules folder. ",
+        "Deno expects the node_modules/ directory to be up to date. ",
+        "Did you forget to run `npm install`?"
```
Comment (bartlomieju) [inline cli/npm/byonm.rs:266]: This is just moved from another place?

Comment (bartlomieju) [reply cli/npm/byonm.rs:266]: We should update the message for Deno 2

File: ext/node/lib.rs [hunk 3]
```diff
@@ -654,6 +655,8 @@ deno_core::extension!(deno_node,
   global_template_middleware = global_template_middleware,
   global_object_middleware = global_object_middleware,
   customizer = |ext: &mut deno_core::Extension| {
+    use ops::vm;
```
Comment (bartlomieju) [inline ext/node/lib.rs:658]: Why move it?

**Review by bartlomieju — APPROVED** [review]

File: ext/node/lib.rs (hunk 3 above)
Comment (dsherret) [reply ext/node/lib.rs]: Reverted.

File: cli/npm/byonm.rs (hunk 2 above)
Comment (dsherret) [reply cli/npm/byonm.rs:266]: Somewhat. It's just a bit smarter now.

**Pull request merged**